Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -1372,6 +1374,43 @@ namespace internal {
         return PMP::compute_face_normal(f, mesh_);
     }
 
+    template <typename FaceRange>
+    void constrain_patch_corners(const FaceRange& face_range)
+    {
+      std::set<vertex_descriptor> visited;
```

Comment:
After all the issues we have had with the inefficiency of `std::set`, do we really want a `std::set`, instead of an `boost::unordered_set` or a `boost::flat_set`?